src/app/Fake.Azure.Kudu/Kudu.fs

```diff
@@ -0,0 +1,122 @@
+﻿/// Contains tasks to stage and deploy Azure website and webjobs using source code deployment with Kudu Sync.
+module Fake.Azure.Kudu
+
+open System
+open System.IO
+#if NETSTANDARD
+open System.Net.Http
+#else
+open System.Net
+#endif
+open Fake.Core
+open Fake.Core.Environment
+open Fake.Core.Trace
```

Comment:
I'd recommend using `Trace.*` instead of opening. We might add `RequireQualifiedAccess` there as well...